src/database/queries.ts

```diff
@@ -5,19 +5,22 @@ import {
   query,
   where,
 } from 'firebase/firestore';
-import { uuid, Vendor, Voucher } from '../types/types';
+import { uuid, Vendor, Voucher, Transaction } from '../types/types';
 import fbApp from './clientApp';
 
 const db = getFirestore(fbApp);
 const testColl = collection(db, 'test-col');
 const vendorCollection = collection(db, 'vendors');
 const voucherCollection = collection(db, 'vouchers');
+const transactionCollection = collection(db, 'transaction');
```

Comment:
'transaction' should be 'transactions' -- this fix should make the queries work.